library/std/src/net/tcp.rs

```diff
@@ -53,6 +53,12 @@ use crate::time::Duration;
 ///     Ok(())
 /// } // the stream is closed here
 /// ```
+///
+/// # Platform-specific Behavior
+///
+/// On Unix writes to the underlying socket in `SOCK_STREAM` mode are made with `MSG_NOSIGNAL` flag.
+/// This suppresses the emission of the  `SIGPIPE` signal when writing to disconnected socket.
+/// In some cases getting a `SIGPIPE` would trigger process termination.
```

Comment:
Two commas:

"On Unix writes ..." -> "On Unix, writes ..."
"In some cases getting a ..." -> "In some cases, getting a ..."

Please also rewrap both paragraphs to 100 chars